clusterloader2/pkg/measurement/util/gatherers/container_resource_gatherer.go

```diff
@@ -42,10 +42,8 @@ const (
 	AllNodes NodesSet = 0
 	// MasterNodes - all containers on Master nodes only
 	MasterNodes NodesSet = 1
```

Comment:
where do we set MasterNodes?

Reply:
I think we don't set it anywhere now. We used to set it for larger clusters when we didn't have MasterAnd* modes...

Reply:
If that is not used anymore, please remove all the code related.

Reply:
I actually think this one is useful - at some point we may not be able to gather even from system components.
I wouldn't remove this one actually.

Reply:
Looks like YAGNI.
I'm fine with either removing this or restoring a flag that could enable it, but not leaving a dead code.

Reply:
ok - good point that it's no longer posisble to set.

Removed

Reply:
Thanks